imjoy/runners/subprocess.py

```diff
@@ -744,6 +744,13 @@ def process_finish(pid=None, cmd=None):
         stop_callback(False, "Plugin process failed to start")
         return False
     # env = os.environ.copy()
+
+    # Replace python to python3
```

Comment:
Is this change required for ngrok? Otherwise I suggest we move it to a different PR.

Reply:
You are right, it's not related.  I was testing a failed plugin.